### PR TITLE
Fix ZeroDivisionError in livecd.py

### PR DESCRIPTION
**Pull request by VladRassokhin**

### CODE REVIEW

**Comment by VladRassokhin** [comment]
anaconda 20.27.3-1 exception report
Traceback (most recent call first):
  File "/usr/lib64/python2.7/site-packages/pyanaconda/sabayon/livecd.py", line 120, in progress
    pct = int(100 \* dest_size / int(self._source_size))
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(_self.__args, *_self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pyanaconda/threads.py", line 168, in run
    threading.Thread.run(self, _args, *_kwargs)
  File "/usr/lib64/python2.7/site-packages/pyanaconda/threads.py", line 76, in get
    self.raise_error(name)
  File "/usr/lib64/python2.7/site-packages/pyanaconda/threads.py", line 85, in wait
    self.get(name).join()
  File "/usr/lib64/python2.7/site-packages/pyanaconda/sabayon/livecd.py", line 173, in install
    threadMgr.wait(THREAD_LIVE_PROGRESS)
  File "/usr/lib64/python2.7/site-packages/pyanaconda/install.py", line 169, in doInstall
    payload.install()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(_self.__args, *_self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pyanaconda/threads.py", line 168, in run
    threading.Thread.run(self, _args, *_kwargs)
ZeroDivisionError: integer division or modulo by zero

**Comment by VladRassokhin** [comment]
Related issue https://bugs.sabayon.org/show_bug.cgi?id=4808

File: pyanaconda/sabayon/livecd.py [hunk 1]
```diff
@@ -117,7 +117,10 @@ def progress(self):
             if dest_size >= self._adj_size:
                 dest_size -= self._adj_size
 
-            pct = int(100 * dest_size / int(self._source_size))
+            if (self._source_size == None || int(self._source_size) == 0):
```
Comment (lxnay) [inline pyanaconda/sabayon/livecd.py]: This is not valid python.
and == should not be used with None.
Why not just checking if source_size is zero...

Comment (VladRassokhin) [inline pyanaconda/sabayon/livecd.py]: Fixed,
BTW int(None) throws exception
I'm not sure self._source_size is not None, so there double check

**Pull request closed without merging**